src/server/raft_client.rs

```diff
@@ -285,7 +285,19 @@ impl Buffer for BatchMessageBuffer {
 
     #[inline]
     fn flush(&mut self, sender: &mut ClientCStreamSender<BatchRaftMessage>) -> grpcio::Result<()> {
-        let batch = mem::take(&mut self.batch);
+        let mut batch = mem::take(&mut self.batch);
+
+        let now = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_nanos() as u64;
+        for msg in batch.msgs.iter_mut() {
+            assert!(now > msg.last_observed_time);
+            let elapsed = nanos_to_secs(now.saturating_sub(msg.last_observed_time));
+            RAFT_MESSAGE_SEND_WAIT_DURATION.observe(elapsed);
```

Comment:
make the metric local

Reply:
The performance result didn't show any difference. Do we really need local metrics?

Reply:
Previously, we encountered global metrics to be a performance impact. As it wouldn't take much effort to make it local, better do it. Check `make_auto_flush_static_metric!`

Reply:
OK.

Reply:
@Connor1996  This job is done. PTAL, thx~